src/main/java/projectzero/cli/Validation.java

```diff
@@ -1,32 +1,24 @@
 package projectzero.cli;
 
-import java.util.Scanner;
+import projectzero.core.UmlClass;
+import projectzero.core.UmlClassManager;
+
+import javax.lang.model.SourceVersion;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 public class Validation {
-    public static boolean isValidMenuInput(int[] validInputList, String input) {
-        boolean validOption = false;
-        try {
-            int option = Integer.parseInt(input);
-            validOption = isValidInt(option, validInputList);
-        } catch (Exception e) {
-            System.out.println("Not valid\n");
-            return false;
-        }
-        return validOption;
-    }
+    public static ArrayList<String> validCommands = new ArrayList<String>(Arrays.asList(
+         "addClass", "addMethod", "addField", "addRelationship", "deleteClass" ,"deleteMethod",
+         "deleteField", "deleteRelationship", "editClass", "editMethod", "editField", "displayAllClasses",
+         "displayClass", "help", "save", "load", "quit"
+    ));
```

Comment:
Should switch this in the future just to an array of Strings. This list in the end will only remain the same size after all commands are predefined.